### PR TITLE
(gce): Support for setting/updating named ports.

### PR DESCRIPTION
**Pull request by jtk54**
Frontend support for setting named ports. Depends on https://github.com/spinnaker/clouddriver/pull/1420.

### CODE REVIEW

File: app/scripts/modules/google/loadBalancer/configure/http/backendService/backendService.component.js [hunk 1]
```diff
@@ -16,6 +16,9 @@ module.exports = angular.module('spinnaker.deck.gce.httpLoadBalancer.backendServ
     controller: function () {
       this.backingData = this.command.backingData;
       this.loadBalancer = this.command.loadBalancer;
+      if (!this.backendService.namedPort) {
+        this.backendService.namedPort = "http";
```
Comment (danielpeach) [inline app/scripts/modules/google/loadBalancer/configure/http/backendService/backendService.component.js]: The double quotes here has probably upset the linter.

File: app/scripts/modules/google/loadBalancer/configure/ssl/gceCreateSslLoadBalancer.controller.ts [hunk 2]
```diff
@@ -216,6 +216,7 @@ class SslLoadBalancerCtrl extends CommonGceLoadBalancerCtrl implements ng.ICompo
   private initializeEditMode (): void {
     this.loadBalancer.portRange = this.loadBalancer.portRange.split('-')[0];
     this.loadBalancer.certificate = this.loadBalancer.certificate.split('/').pop();
+    this.loadBalancer.backendService.namedPort = this.loadBalancer.backendService.namedPort ? this.loadBalancer.backendService.namedPort : 'http';
```
Comment (danielpeach) [inline app/scripts/modules/google/loadBalancer/configure/ssl/gceCreateSslLoadBalancer.controller.ts]: In JS, you can use `||` where you would use `?:` in groovy.

**Comment by danielpeach** [comment]
I would hold off on merging this until you have a corresponding PR for configuring the port name mapping in the server group wizard (and in Clouddriver), since someone who created a load balancer with a port name other than `http` would not be able to use it right now.

**Comment by jtk54** [comment]
This feature is on hold for now, closing.

**Pull request closed without merging**